modules/service-bus/namespaces/authorization-rules/README.md

```diff
@@ -1,4 +1,4 @@
-# ServiceBus Namespace Authorization Rules `[Microsoft.ServiceBus/namespaces/authorizationRules]`
+# ServiceBus Namespace Authorization Rules `[Microsoft.ServiceBus/namespaces/AuthorizationRules]`
```

Comment:
Should this be camelCase?

Reply:
Hey @lsnoddy, yes, to align with the resource provider documentation: https://learn.microsoft.com/en-us/azure/templates/microsoft.servicebus/namespaces/authorizationrules?pivots=deployment-language-bicep#resource-format